_docs/reference/release-notes.md

```diff
@@ -8,4 +8,17 @@ layout: docs
 type: markdown
 ---
 
-TBD
+## Istio v0.1
+
+Istio v0.1 is the initial release of Istio. It works in a single Kubernetes cluster with a small number of services. Istio v0.1 supports the following features:
+- Installation of Istio into a Kubernetes namespace with a single command.
+- Injection of Envoy proxies into Kubernetes pods using `istioctl kube-inject`.
+- Automatic traffic capture for Kubernetes pods using iptables.
+- In-cluster load balancing for HTTP, gRPC, and TCP traffic.
+- Basic Kubernetes Ingress and Egress support.
```

Comment:
What about the route rules, resilience features? Standard ingress does not have circuit breakers or retries with timeouts?

All of the routing support is available at ingress including traffic split, a/b tests based on headers without annotation nonsense, resilience features like timeours